app/webhooks/handlers.py

```diff
@@ -24,7 +25,14 @@ async def webhook_handler(payload: GenericJsonPayload, integration=None, webhook
     # Check if a filter is present in the filtered data
     for data in transformed_data:
         if data.get("status", "OK") != 'OK':
-            logger.info(f"'{data}' point received was filtered")
+            logger.warning(f"'{data}' point received was filtered")
```

Comment:
@vgarcia13 how many of these do you think we'll see?

Reply:
Not sure how many generic webhook configs we have today, but for the `Namibia Waldeck Rhino Project` webhook we have filtered 5 since the new JQ filter was implemented.